emoji/src/test/java/com/vanniktech/emoji/EmojiManagerTest.java

```diff
@@ -49,10 +47,61 @@
     assertThat(EmojiManager.getInstance().getCategories()).isNotEmpty();
   }
 
+  @Test
+  public void noProviderInstalled() {
```

Comment:
Let's make it for now package private so it can't be used. I anyways want the install() method to provide some type on which you can do a bunch of strings (like the string queries that I proposed) and this method might get added too.